config-templates/redhat/odf-local/changelog.json

```diff
@@ -1,4 +1,18 @@
 [
+    {
+        "4.13": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "24 July 2023",
+                "cves": "",
+                "golang": "1.20.5",
+                "ubi": "8.8-860",
```

Comment:
is it still based on old ubi image and not on `8.8-1014`?

Reply:
yes, new patch release for all template is planning for next week, which will include all VA fixes